the-exterminators/error-annihilator
# Project overview textfix

## Linked Issues

## PR Description
I changed the way the search bar functions, but in the frontend it has barely any change. 
Isabelle and I created the Project Overview and Project Single Views, where you can see tickets belonging to a project.

## Changes


## How to test

1. fetch branch
2. Open, build and run project
3. Check if project builds and looks correct on first sight
4. Navigate to Project Overview
5. Here there should be a list of all projects
6. Once you click on a project, it opens up into a page with the title, description and all tickets belonging to that project.
7. When you click on a ticket, it opens up a viewer/editor

### Expected behaviour:

## Checklist author


## Checklist reviewer/test-engineer
